wurst/extensions/PlayerExtensions.wurst

```diff
@@ -18,3 +18,9 @@ public function player.setHandicapXP(real handicap)
 
 public function player.setColor(playercolor color, bool changeExisting)
     SetPlayerColorBJ(this, color, changeExisting)
+
+public function player.isObserver() returns bool
+    return IsPlayerObserver(this)
+
+public function player.getHero() returns unit

```

Comment:
This makes more sense as `getTroll`, especially if a player using Beast Master has multiple heroes in the future.

Reply:
@jonathanmclaus 👍